src/SDL_gpu_shadercross.c

```diff
@@ -312,9 +297,18 @@ static void *SDL_ShaderCross_INTERNAL_CompileUsingDXC(
     /* Non-static DxcInstance, since the functions we call on it are not thread-safe */
     IDxcCompiler3 *dxcInstance = NULL;
 
-    SDL_DxcCreateInstance(&CLSID_DxcCompiler,
-                          IID_IDxcCompiler3,
-                          (void **)&dxcInstance);
+    #if defined(SDL_PLATFORM_XBOXONE) || defined(SDL_PLATFORM_XBOXSERIES)
+    SDL_DxcCreateInstance(
```

Comment:
Since this can be null maybe assert that it's non-null so that it fails in a coherent way if you didn't init successfully first?
I see the old LogErrors about this seem to have been removed too.

Reply:
I guess it could if the client forgot to call Init